### ISSUE TITLE
Feature Request: preventing loading / unloading systems

### ISSUE THREAD

**Issue by elsassph** [pixijs/pixijs#10393]
### Description

It doesn't seem to be possible to prevent the `EventSystem` and `AccessibilitySystems` to load and initialize.

I tried removing the extensions early, but they still initialize and the mixins definitely are always applied:
```js
extensions.remove(EventSystem);
extensions.remove(AccessibilitySystem);
```

**Comment by bigtimebuddy** [comment]
Here's an example to illustrate it a little clearer: https://jsfiddle.net/bigtimebuddy/enhujtbr/

When you `remove()` the extension is it removing the Renderer's system from getting installed when the Renderer is created. It does not, however, prevent the installation of the mixins on DisplayObjects. In order to achieve that you need to do something like we discussed here #10392 and not import the code.

**Comment by elsassph** [comment]
Indeed, and I raised 2 issues because extensions and "renderables" are different topics.

As I answered on #10392 short of patching a number of files, extensions always initialize.

Potentially the extensions could "initialize late", and the mixins would also be applied late. It could be automatic when you first create a Pixi `Application`, or allow explicitly calling an extensions init function if that'd be needed. This would allow removing an extension before it has side effects.
